pkg/genericapiserver/config.go

```diff
@@ -399,7 +399,9 @@ func (c *Config) ApplyDelegatingAuthenticationOptions(o *options.DelegatingAuthe
 	}
 
 	c.Authenticator = authenticator
-	c.OpenAPIConfig.SecurityDefinitions = securityDefinitions
+	if c.OpenAPIConfig != nil {
```

Comment:
Could we go via `Config.Complete` to fill the `SecurityDefinitions` if still nil?